sound/soc/sof/imx/imx8.c

```diff
@@ -416,6 +416,13 @@ struct snd_sof_dsp_ops sof_imx8_ops = {
 	/* DAI drivers */
 	.drv = imx8_dai,
 	.num_drv = 1, /* we have only 1 ESAI interface on i.MX8 */
+
+	/* ALSA HW info flags */
+	.hw_info =	SNDRV_PCM_INFO_MMAP |
+			SNDRV_PCM_INFO_MMAP_VALID |
+			SNDRV_PCM_INFO_INTERLEAVED |
+			SNDRV_PCM_INFO_PAUSE |
+			SNDRV_PCM_INFO_NO_PERIOD_WAKEUP
```

Comment:
I already see this code in topic/sof-dev?

Reply:
@plbossart, I thought the same thing *but* actually we have two dsp_ops in imx8.c. One for imx8x and one for imx8.
 
Same as you I could swear that this code was already in topic/sof-dev. Will triple check tommorrow.

Reply:
oh right, indeed there's one missing.